### PR TITLE
Integrations docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/deployment/bots.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ - Hubot adapter for Mattermost written in Javascript that uses Web API and Websockets, developed by Andy Lo-A-Foe.
+ - Use the bot to listen for commands and to execute actions based on your requests.
+ - Invite your bot to any Mattermost channel just like a regular user.
+ - Source code + docs: https://github.com/loafoe/hubot-matteruser
+
+Python Chatbot
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - A python based chat bot for Mattermost, developed by Alex Tzonkov.
+ - **XXXXXX**
```
Comment (amyblais) [inline source/deployment/bots.rst]: @aaronrothschild Wondering if you can help with more details on this bot as I don't quite understand the features.

File: source/guides/administrator.rst [hunk 2]
```diff
@@ -169,6 +169,10 @@ Learn how to extend Mattermost by integrating your workflows.
    /developer/toolkit*
    /deployment/atlassian-integrations*
    /deployment/video-and-audio-calling*
+   /deployment/ci-cd-tools*
+   /deployment/incident-response-and-monitoring*
+   /deployment/bots*
```
Comment (jasonblais) [inline source/guides/administrator.rst]: Let's move `video-and-audio-calling` to below bots.

File: source/deployment/bots.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+ - Welcome users to your Mattermost instance, developed by Mattermost.
+ - Add a Welcome Bot that helps add new team members to channels to improve onboarding and HR processes.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-welcomebot
+
+BotKube
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Monitor and debug Kubernetes resources, developed by InfraCloud Technologies.
+ - Helps you monitor your Kubernetes cluster, debug critical deployments and get recommendations for standard practices.
```
Comment (jasonblais) [inline source/deployment/bots.rst]:
```suggestion
 - Monitor your Kubernetes cluster, debug critical deployments and get recommendations for standard practices.
```

File: source/deployment/ci-cd-tools.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ - Supports GitHub Enterprise, works with SaaS and Enterprise versions of GitHub.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-github 
+
+GitLab
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Two-way integration between Mattermost and GitLab, developed by Romain Maneschi, supported by Mattermost.
+ - Get reminders on issues and merge requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests and comments.
+ - Supports GitLab On Premise, works with SaaS and On Premise versions of GitLab.
```
Comment (jasonblais) [inline source/deployment/ci-cd-tools.rst]:
```suggestion
 - Supports SaaS and On Premise versions of GitLab.
```

File: source/deployment/ci-cd-tools.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-gitlab
+
+Jenkins
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Connect your Jenkins account to Mattermost, developed by Jenkins.
+ - Sends webhook notifications from Jenkins to Mattermost.
+ - Source code + docs: https://github.com/jenkinsci/mattermost-plugin
+
+Jenkins Plugin
```
Comment (jasonblais) [inline source/deployment/ci-cd-tools.rst]: Let's move this above the "Jenkins" integration

File: source/deployment/ci-cd-tools.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+ - Get reminders on issues and merge requests that need your attention.
+ - Get notifications in Mattermost about mentions, review requests and comments.
+ - Supports GitLab On Premise, works with SaaS and On Premise versions of GitLab.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-gitlab
+
+Jenkins
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Connect your Jenkins account to Mattermost, developed by Jenkins.
+ - Sends webhook notifications from Jenkins to Mattermost.
```
Comment (jasonblais) [inline source/deployment/ci-cd-tools.rst]:
```suggestion
 - Send webhook notifications from Jenkins to Mattermost channels.
```

File: source/deployment/ci-cd-tools.rst [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+ - Source code + docs: https://github.com/jenkinsci/mattermost-plugin
+
+Jenkins Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Two-way integration between Mattermost and Jenkins, developed by Wasim Thabraze, supported by Mattermost.
+ - Enables interacting with jobs and builds via slash commands in Mattermost.
+ - Abort builds, enable and disable jobs, and get test results and build logs.
```
Comment (jasonblais) [inline source/deployment/ci-cd-tools.rst]:
```suggestion
 - Abort builds, enable and disable jobs, get test results, and build logs.
```

File: source/deployment/incident-response-and-monitoring.rst [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect incident response and monitoring tools in Mattermost.
+
+Here are some popular options below, including self-hosted on-prem or self-hosted private cloud solutions, and vendor-hosted SaaS solutions.
+
+AWS CloudWatch SNS
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Two-way integration between Mattermost and Amazon AWS SNS, developed by Carlos Tadeu Panato Junior, supported by Mattermost.
+ - Receive SNS notifications from Alerts created by Amazon AWS CloudWatch and sent via AWS SNS.
+ - Get operational data and actionable insights to monitor Mattermost in order to respond to system-wide performance changes.
```
Comment (jasonblais) [inline source/deployment/incident-response-and-monitoring.rst]: Is it 'performance changes'?

File: source/deployment/incident-response-and-monitoring.rst [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-aws-SNS
+
+PagerDuty
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Get automatic information regarding the state of an incident to Mattermost, developed by PagerDuty.
+ - Receive all incident updates for any PagerDuty Service in any Mattermost channel to optimize incident resolution time and process.
```
Comment (jasonblais) [inline source/deployment/incident-response-and-monitoring.rst]:
```suggestion
 - Send incident updates for any PagerDuty Service to any Mattermost channel to optimize incident resolution time and process.
```

File: source/deployment/incident-response-and-monitoring.rst [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+ - Docs: https://www.pagerduty.com/docs/guides/mattermost-integration-guide/
+
+Splunk
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Monitor a Mattermost environment.
+ - Receive application performance insights in any Mattermost channel for better troubleshooting and monitoring and to increase uptime.
```
Comment (jasonblais) [inline source/deployment/incident-response-and-monitoring.rst]:
```suggestion
 - Send application performance insights to any Mattermost channel for better troubleshooting and monitoring, and to increase system uptime.
```

File: source/deployment/productivity-tools.rst [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect productivity tools in Mattermost.
+
+Here are some popular options below, including self-hosted on-prem or self-hosted private cloud solutions, and vendor-hosted SaaS solutions.
+
+Matterpoll for polls
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Create a poll by using a slash command, developed by Hanzei and Yusuke Nemoto.
+ - Polls can be created in different supported languages, currently French, German and Japanese.
```
Comment (jasonblais) [inline source/deployment/productivity-tools.rst]:
```suggestion
 - Polls can be created in different languages, including English, French, German and Japanese.
```

File: source/deployment/productivity-tools.rst [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+ - Source code + docs: https://github.com/matterpoll/matterpoll
+
+Google Calendar
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Time management tool, developed by Wasim Thabraze.
```
Comment (jasonblais) [inline source/deployment/productivity-tools.rst]:
```suggestion
 - Time management tool for Google Calendar, developed by Wasim Thabraze.
```

File: source/deployment/productivity-tools.rst [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+ - Get reminders about appointments and meetings from configured Google Calendar to any Mattermost channel.
```
Comment (jasonblais) [inline source/deployment/productivity-tools.rst]:
```suggestion
 - Get reminders about appointments and meetings from a configured Google Calendar to any Mattermost channel.
```

File: source/deployment/productivity-tools.rst [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Tool for screen recording, screenshots, and GIF creation in the cloud, developed by CloudApp team.
+ - Record a screen, generate a GIF or create a screenshot, and share in Mattermost.
+ - Docs: https://www.getcloudapp.com/integrations/mattermost
+
+Gyazo
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Screenshot app, developed by Sean Sackowitz.
```
Comment (jasonblais) [inline source/deployment/productivity-tools.rst]:
```suggestion
 - Screenshot application, developed by Sean Sackowitz.
```

File: source/deployment/productivity-tools.rst [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+ - Take screenshots, screen GIFs and full page captures, and share in Mattermost.
+ - Source code: https://github.com/seansackowitz/mattermost-plugin-gyazo
+
+Autotranslation Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Two-way integration between Amazon AWS Translate and Mattermost, developed by Mattermost.
```
Comment (jasonblais) [inline source/deployment/productivity-tools.rst]: This may be a one-way integration, not sure

Comment (aaronrothschild) [reply source/deployment/productivity-tools.rst]: changed the text to be ambiguous

File: source/deployment/incident-response-and-monitoring.rst (hunk 8 above)
Comment (aaronrothschild) [reply source/deployment/incident-response-and-monitoring.rst]: Changed text to become "performance changes system-wide"

**Pull request merged**